web-admin/src/features/organizations/settings/OrgNameSettings.svelte

```diff
@@ -119,6 +119,7 @@
       description={`Your org URL will be https://ui.rilldata.com/${sanitizeOrgName($form.name)}, to comply with our naming rules.`}
       textClass="text-sm"
       alwaysShowError
+      width="520px"
```

Comment:
Looks like this breaks responsiveness. Switch it to `maxWidth`?
<img width="846" alt="image" src="https://github.com/user-attachments/assets/5cb5314f-6ecf-411b-998e-1e574e6787e7">
<img width="843" alt="image" src="https://github.com/user-attachments/assets/7a2c464b-bd30-41f5-9967-4d2f0f1408b5">